src/execute/ExecuteOutlineMode.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Box } from "@chakra-ui/react";
+import { Outline } from "common/components/outline/Outline";
+import { UIBlogArticle } from "domain/blog-article/UIBlogArticle";
+import { UIOutline } from "domain/common/UIOutline";
+import React from "react";
+
+type Props = {
+  exerciseXML: string;
+  growthExerciseId: string | undefined;
+  handleOutlineUpdate: (uiOutline: UIOutline) => void;
+};
+
+export const ExecuteOutlineMode: React.FC<Props> = ({
```

Comment:
What dont we name this OutlineMode instead, any specific reason to prepend Execute

Reply:
Done